snippets/common.yml

```diff
@@ -230,7 +275,14 @@ contacts:
     react_native:
       code: '@todo'
     ios:
-      code: '@todo'
+      code: |
+            NSError *error;
+            QueryOptions *options;
```

Comment:
This needs to be initialized, it's just a null pointer like this, will lead to runtime crash